ClosedXML_Tests/Excel/Saving/SavingTests.cs

```diff
@@ -254,5 +254,20 @@ public void CanSaveFileWithPictureAndComment()
                 wb.SaveAs(ms);
             }
         }
+
+        [Test]
+        public void PreserveChartsWhenSaving()
```

Comment:
What do you think about adding another test for the opposite - say, delete all pictures and make sure no empty Drawing.xml is presented?

Reply:
Added new test below this one.